sunpy/timeseries/sources/eve.py

```diff
@@ -57,32 +57,48 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, title="EVE/ESP Level 1", **kwargs):
         """
+        Plots the EVE ESP Level 1 timeseries data.
+
         Parameters
         ----------
         title : `str`, optional
-            Plot title.
+            The title of the plot. Defaults to "EVE/ESP Level 1".
         **kwargs : `dict`
             Additional plot keyword arguments that are handed to `pandas.DataFrame.plot`.
-        """
 
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
+            The plot axes.
+        """
         self._validate_data_for_plotting()
-
-        names = ('Flux \n 0.1-7nm', 'Flux \n 18nm', 'Flux \n 26nm', 'Flux \n 30nm', 'Flux \n 36nm')
-
+        names = ("Flux \n 0.1-7nm", "Flux \n 18nm", "Flux \n 26nm", "Flux \n 30nm", "Flux \n 36nm")
         axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
-
         axes[-1].set_xlim(self.to_dataframe().index[0], self.to_dataframe().index[-1])
-
         axes[0].set_title(title)
         for i, ax in enumerate(axes):
             ax.set_ylabel(names[i])
-            ax.legend(loc='upper right')
-        axes[-1].set_xlabel('Time (UT) ' + str(self.to_dataframe().index[0])[0:11])
-        axes[-1].xaxis.set_major_formatter(dates.DateFormatter('%H:%M'))
+            ax.legend(loc="upper right")
+        axes[-1].set_xlabel("Time (UT) " + str(self.to_dataframe().index[0])[0:11])
+        axes[-1].xaxis.set_major_formatter(dates.DateFormatter("%H:%M"))
+        return axes
 
+    @peek_show
+    def peek(self, title="EVE/ESP Level 1", **kwargs):
```

Comment:
Does peek need to take title?

Reply:
I don't see an issue here right? Do we not want that?

Reply:
Why would we want it on peek?

Reply:
I do wonder if we want it on plot as well.

Reply:
I think at the very least, peek shouldn't take any arguments.

Reply:
Lets leave this for now (we shouldn't change the signature without deprecating anyway), we can always revisit the signature of `peek()` in another issue or PR.